e2e/scripts/patches/focal.patch

```diff
@@ -20,20 +20,22 @@ index ccf213e0..40a2aa9f 100644
                 dbus,
                 libdbus-1-dev,
 diff --git a/debian/rules b/debian/rules
-index 43646c6a..0708aa3d 100755
+index f9705a3d..3513e53f 100755
 --- a/debian/rules
 +++ b/debian/rules
-@@ -25,6 +25,9 @@ export DH_GOLANG_INSTALL_ALL := 1
+@@ -25,9 +25,8 @@ export DH_GOLANG_INSTALL_ALL := 1
  # Tests needing sudo will be skipped automatically
  export ADSYS_SKIP_INTEGRATION_TESTS=1
 
+-# We want to take whatever ubuntu propose to us (as it won’t download a newer version),
+-# as long as it matches the go.mod go stenza which is the language requirement.
+-export GOTOOLCHAIN := local
```

Comment:
you still want to keep GOTOOLCHAIN = local in those diff, or they will be forceably upgraded.

Reply:
That was on purpose. If we don't upgrade the Go version, we can't build the package in Focal.

Reply:
Right, but even here, we want to take whatever is available locally, or it will try to bump to align with the go and toolchain stenza in `go.mod`.